Newspack/Newspack/Controllers/AssetsViewController.swift

```diff
@@ -0,0 +1,39 @@
+import UIKit
+import WordPressFlux
+
+class AssetsViewController: UITableViewController {
+
+    var items = [URL]()
+    var receipt: Any?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        items = StoreContainer.shared.assetStore.listAssets()
+
+        receipt = StoreContainer.shared.folderStore.onChange { [weak self] in
+            self?.items = StoreContainer.shared.assetStore.listAssets()
+            self?.tableView.reloadData()
+        }
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return items.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "AssetCell", for: indexPath)
```

Comment:
[This trick for the Cell ID](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/Extensions/UITableViewCell+Helpers.swift#L11) might become handy (or perhaps an overkill!!)

Reply:
Nice! Thanks for the tip :)